bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -275,6 +290,13 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
                     logger.error("The url title is greater than " + API_MAX_URL_TITLE_LENGTH + " characters.");
                     return false;
                 }
+            } else if (StringUtils.isEmpty(urlTitle) && !StringUtils.isEmpty(defaultUrlTitle)) {
```

Comment:
The part `StringUtils.isEmpty(urlTitle) && ` is unneeded and would read better if it were removed.